projects/plugins/jetpack/extensions/blocks/goodreads/goodreads.php

```diff
@@ -33,13 +33,16 @@ function register_block() {
  * @return string
  */
 function load_assets( $attr ) {
-	/*
-	 * Enqueue necessary scripts and styles.
-	 */
 	Jetpack_Gutenberg::load_assets_as_required( __DIR__ );
 
-	if ( isset( $attr['link'] ) ) {
-		wp_enqueue_script( 'goodreads-block', $attr['link'], array(), JETPACK__VERSION, true );
+	if ( isset( $attr['link'] ) && isset( $attr['id'] ) ) {
+		wp_enqueue_script(
+			'jetpack-goodreads-' . $attr['id'],
+			$attr['link'],
```

Comment:
I would recommend escaping those 2 values, for extra safety.

```suggestion
			'jetpack-goodreads-' . esc_attr( $attr['id'] ),
			esc_url( $attr['link'] ),
```

Reply:
Makes sense, thanks!

Reply:
Actually, I just tested this, and it breaks the attribute. I've switched to `esc_url_raw` which doesn't - not an expert on this, but that should be ok?